src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -152,6 +159,7 @@ internal sealed override ImmutableArray<string> NotNullWhenFalseMembers
             _property = property;
             _isAutoPropertyAccessor = false;
             _isExpressionBodied = true;
+            _containsFieldKeyword = NodeContainsFieldKeyword(syntax);
```

Comment:
>NodeContainsFieldKeyword(syntax); [](http://example.com/codeflow?start=36&length=33)

Do we support field keywords in indexers? Fine to add a prototype comment.

Reply:
No, we don't support them. Same answer as in https://github.com/dotnet/roslyn/pull/57076#discussion_r774923711.